docs/guides/MONITORING_QUICKSTART.md

```
@@ -0,0 +1,828 @@
+  - 자동 대시보드 배포 시스템 구축
+  - Prometheus 데이터 소스 자동 설정
+
+- ✅ **5개의 대시보드 배포**
+  1. Docker + System Dashboard
+  2. Django Prometheus
+  3. PostgreSQL Database
+  4. Redis Dashboard
+  5. Node Exporter Full
+
+### 접속 방법
+- **Grafana UI**: http://localhost:3000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)